types/index.d.ts

```diff
@@ -227,6 +227,12 @@ declare namespace Realm {
      */
     type SortDescriptor = [string] | [string, boolean];
 
+    /**
+     * Dictionary
+     * @see { @link https://realm.io/docs/javascript/latest/api/Realm.Dictionary.html }
```

Comment:
I don't believe this exists yet?

Reply:
True. We will need to update the API doc too.